File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/magic1/p2.c
#include<stdio.h>
void main(){

	int num;
	int rem=0;
	int count=0;

	printf("Enter the number:");
	scanf("%d",&num);

	for (int i=1;i<=num;i++){
		rem=num%10;
		count=count+1;
		num=num/10;
	}
	printf("count is:%d\n,count");
}

File: java-main/java-main/LeetCodes/p2.java
import java.util.*;

class Reverse{
	public static void main(String[] args){

		Scanner sc= new Scanner(System.in);

		System.out.println("Enter number");
		int num=sc.nextInt();

		int rev=Integer.MIN_VALUE;
	
		while(num !=0){

			int rem=num%10;
			rev=rev*10+rem;
			num=num/10;
		}
		System.out.println(rev);
	}
}

